examples/multisig_plain/lib.rs

```diff
@@ -486,7 +486,7 @@ mod multisig_plain {
         ///
         /// Its return value indicates whether the called transaction was successful.
         /// This can be called by anyone.
-        #[ink(message)]
+        #[ink(message, payable)]
```

Comment:
Hmm, I just thought a bit more about this:

This covers the case where `Transaction.transferred_value` has been set. The contract currently supports setting this value, but if it is set the contract would pay for it out of its own fund.

This is not really a proper fix, since the contract still doesn't check if the funds transferred with the transaction are `==` to `Transaction.transferred_value`

I would create a follow-up ticket to implement this properly. In a proper implementation the `submit_transaction` would require `transferred_value` to be transferred along with the transaction submission.